src/test/java/com/varchenko/FibonacciSequenceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.varchenko;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import static com.varchenko.FibonacciSequence.fib;
+
+public class FibonacciSequenceTest {
+    FibonacciSequence fibonacciSequence = new FibonacciSequence() ;
+        @Test
+        void Fibonacci_numbersFrom0to10() {
+            for (int index = 0; index < 10; index++) {
+
+                int expectedResult = fib(index);
```

Comment:
You have to set the expected value by yourself.
Right now, you just test that your method returns the same value each time.

Reply:
done